.github/workflows/bokeh-ci.yml

```diff
@@ -93,6 +93,29 @@ jobs:
           mypy bokeh
           mypy release
 
+      - name: Get Changed Files
+        id: get_changed_files
+        # include this action to filter for changed files only
+        # (instead of checking entire docs every time)
+        uses: lots0logs/gh-action-get-changed-files@2.1.4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Vale
+        uses: errata-ai/vale-action@v1.3.0
+        with:
+          # include remotely maintained "Google" style
+          styles: |
+            https://github.com/errata-ai/Google/releases/latest/download/Google.zip
+
+          # # only check modified lines, instead of all docs every time (feature currently broken)
```

Comment:
extra `#`

Reply:
If its currently checking all of the files, then it's already very fast and I'd suggest we just check all the files all the time.

Reply:
So far no files have been checked - I first of all wanted to see, if Vale builds and executes at all in the CI... I'll test checking some (and also all files) soon!